package.json

```diff
@@ -1,98 +1,91 @@
-    "test:coverage": "node --max-old-space-size=8192 ./node_modules/.bin/truffle run coverage",
-    "test:ganache:mainnet": "truffle exec ./test-integration/main.js --network ganache-mainnet",
-    "format:prettier": "prettier --write contracts/**/**/**/**/**/**.sol",
-    "format:prettier:print": "prettier contracts/**/**/**/**/**/**.sol",
-    "verify:ropsten": "truffle run verify TDAI TUSDC Settings ATMSettings MarketsState EscrowFactory MarketFactory LogicVersionsRegistry Escrow Lenders EtherCollateralLoans TokenCollateralLoans LendingPool InterestConsensus LoanTermsConsensus ATMFactory ATMGovernance TLRToken ChainlinkAggregator --network ropsten",
-    "verify:mainnet": "truffle run verify TDAI TUSDC Settings ATMSettings MarketsState EscrowFactory MarketFactory LogicVersionsRegistry Escrow Lenders EtherCollateralLoans TokenCollateralLoans LendingPool InterestConsensus LoanTermsConsensus ATMFactory ATMGovernance TLRToken ChainlinkAggregator --network mainnet",
-    "verify:rinkeby": "truffle run verify TDAI TUSDC Settings ATMSettings MarketsState EscrowFactory MarketFactory LogicVersionsRegistry Escrow Lenders EtherCollateralLoans TokenCollateralLoans LendingPool InterestConsensus LoanTermsConsensus ATMFactory ATMGovernance TLRToken ChainlinkAggregator --network rinkeby"
-  },
-  "devDependencies": {
-    "@gnosis.pm/mock-contract": "^3.0.7",
-    "@typechain/truffle-v5": "^3.0.0",
-    "@types/chai": "^4.2.14",
-    "@types/expect": "^24.3.0",
-    "@types/mocha": "^8.0.4",
-    "axios": "^0.21.1",
-    "bl": "^2.2.1",
-    "chai": "^4.2.0",
-    "dotenv": "^8.2.0",
-    "eslint": "^5.16.0",
-    "eth-gas-reporter": "^0.2.15",
-    "ethereumjs-abi": "^0.6.8",
-    "ethereumjs-util": "^6.1.0",
-    "ganache-cli": "^6.4.4",
-    "ganache-time-traveler": "^1.0.5",
-    "husky": "^4.2.3",
-    "jsonfile": "^6.0.1",
-    "leche": "^2.3.0",
-    "lint-staged": "^10.5.3",
-    "lodash": "^4.17.19",
-    "mocha": "^7.1.0",
-    "node-fetch": "^2.6.1",
-    "prettier": "^2.0.2",
-    "prettier-plugin-solidity": "^1.0.0-alpha.46",
-    "serialize-javascript": "^3.1.0",
-    "solhint": "^2.1.0",
-    "solhint-plugin-prettier": "^0.0.4",
-    "solidity-coverage": "^0.7.9",
-    "truffle": "5.1.41",
-    "truffle-assertions": "^0.9.2",
-    "truffle-plugin-verify": "^0.3.10",
-    "ts-node": "^9.0.0",
-    "typechain": "^4.0.0",
-    "typescript": "^4.1.2",
-    "yargs": "^15.3.1"
-  },
-  "dependencies": {
-    "@chainlink/contracts": "^0.0.10",
-    "@openzeppelin/contracts": "2.5.1",
-    "@openzeppelin/contracts-ethereum-package": "2.5.0",
-    "@openzeppelin/upgrades": "^2.8.0",
-    "bignumber.js": "^9.0.0",
-    "elliptic": "^6.5.3",
-    "ethers": "^5.0.12",
-    "ganache-core": "^2.11.2",
-    "truffle-hdwallet-provider": "^1.0.17",
-    "web3": "1.0.0-beta.55",
-    "web3-provider-engine": "15.0.0",
-    "web3-utils": "1.0.0-beta.55"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
+    "name": "teller-protocol-v1",
```

Comment:
what happened here? lets leave it as 2 spaces

Reply:
2 spaces for all of JS? Sure